python/pythonmonkey/builtin_modules/console.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file     console.js
+ *           Temporary implementation of console.log etc.
+ * @author   Wes Garland, wes@distributive.network
+ * @date     June 2023
+ */
+function Console(print)
+{
+  this.log   = print;
+  this.debug = print;
+  this.error = print;
+  this.warn  = print;
```

Comment:
Shouldn't `console.error` and `console.warn` output to `stderr`?

Reply:
The will someday. I think you have a better console module in the works?  This one was only designed to make sure there is water flowing through the "builtin_modules" pipe.  It doesn't even colourize output.